src/transformers/file_utils.py

```diff
@@ -1211,7 +1213,7 @@ def http_user_agent(user_agent: Union[Dict, str, None] = None) -> str:
     """
     Formats a user-agent string with basic info about a request.
     """
-    ua = "transformers/{}; python/{}".format(__version__, sys.version.split()[0])
+    ua = f"transformers/{__version__}; python/{sys.version.split()[0]}; session_id/{SESSION_ID}"
```

Comment:
can we trim the id slightly to e.g. 12 chars? @philschmid cc @n1t0

Reply:
I don't have a strong opinion on this

Reply:
I wanted to leave this unchanged and adding 20 more characters to it won`t be that big of a problem. But I am open to adjusting it.